x/concentrated-liquidity/lp_test.go

```diff
@@ -5,47 +5,62 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	concentrated_liquidity "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity"
```

Comment:
the standard we seem to have been following is cl, but happy to switch to this if you feel the verbosity makes things clearer (would mean clmodel would need to be concentrated_liquidity_model though etc)

Reply:
I'm down for cl more than this., I think it was auto-applied. Changed now